almost-jira/src/test/java/pl/edu/pw/elka/pap/z16/almostjira/controllers/TasksControllerUnitTest.java

```diff
@@ -0,0 +1,40 @@
+package pl.edu.pw.elka.pap.z16.almostjira.controllers;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.springframework.http.HttpStatus;
+import pl.edu.pw.elka.pap.z16.almostjira.controllers.tasks.TasksController;
+import pl.edu.pw.elka.pap.z16.almostjira.services.ProjectService;
+import org.mockito.Mock;
+
+import java.util.Map;
+
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.when;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@ExtendWith(MockitoExtension.class)
+class TasksControllerUnitTest {
```

Comment:
I think that we could add general class that will store all configurations like @ExtendWith(MockitoExtension.class) that will be extended by every test class, but maybe it should be done in another issue